examples/flutter_gallery/.idea/workspace.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is the minimal workspace.xml file we added to the template to trigger `main.dart` being shown in the editor. The contents are identical to:
https://github.com/flutter/flutter/blob/master/packages/flutter_tools/templates/create/.idea/workspace.xml.tmpl